templates/web/panels/panel.tt.html

```diff
@@ -1,6 +1,65 @@
-[% panel = panels.$panel_id %]
-<!-- start templates/[% template.name %] - panel_id: [% panel_id %] -->
-[% IF (panel.type == "card") OR (panel.type == "inline") %]
+[%
+   # check that the panel_id arg is present
+   THROW "panel.tt.html was called without a panel_id arg" IF ! panel_id;
+
+   # preamble: build a map of evaluation values to presentation features
+   SET show_evaluation = {	good 	 => {img_color => 'green',  icon => 'check', td_color => 'green' },
+        					bad  	 => {img_color => 'red',    icon => 'cancel',td_color => 'red'   },
+        					average  => {img_color => 'orange', icon => 'check', td_color => ''      },
+        					neutral  => {img_color => 'grey',   icon => 'check', td_color => 'orange'},
+        					unknown  => {img_color => 'grey',   icon => 'help'}, td_color => 'grey'  };
```

Comment:
Why don't we put it in the perl code ?

Reply:
I guess we put the generation of data in the Perl code, but we leave everything that is a presentation choice to the template